src/cmd/meta.rs

```diff
@@ -55,6 +55,14 @@ pub enum Kubie {
         /// Exit early if a command fails when using a wildcard context.
         #[structopt(short = "e", long = "exit-early")]
         exit_early: bool,
+        /// Always print context headers in output, even if not a TTY.
+        /// Overrides behavior.print_context_in_exec in Kubie settings file.
+        #[structopt(long = "context-headers")]
+        override_context_headers: bool,
+        /// Never print context headers in output.
+        /// Overrides behavior.print_context_in_exec in Kubie settings file.
+        #[structopt(long = "no-context-headers", conflicts_with = "override-context-headers")]
+        override_no_context_headers: bool,
```

Comment:
Maybe to help with the [bit here](https://github.com/sbstp/kubie/pull/72/files#r602452806) these should be another enum with states like `ForcePrint`, `ForceNoPrint`, and `Unset`? I'm just not quite sure how to implement that... still giving it a think. Let me know your thoughts @sbstp...

Reply:
What about instead of `--context-headers` and `--no-context-headers`, we were to do something like:

`--context-headers=value`, where the value can be one of `auto`, `always`, `never`, just like the settings from the config file? I'd imagine the flag would override the config.

Reply:
I ended up implementing this idea in [this commit](https://github.com/sbstp/kubie/pull/72/commits/49be9dc63d944b9f08ebe7a9d67f4152ae31da15)... see if you like it... if not it's easily revertable. :)